server/src/test/java/org/apache/druid/metadata/SQLMetadataConnectorTest.java

```diff
@@ -305,6 +303,140 @@ public void testIsTransientException()
     );
   }
 
+  @Test
+  public void test_useShortIndexNames_true_tableIndices_areNotAdded_ifExist()
+  {
+    tablesConfig = new MetadataStorageTablesConfig(
+        "druidTest_base_table_name_true",
```

Comment:
Let's use a shorter base name here to aid with the next comments:
```suggestion
        "druidTest",
```